FreydCategoriesForCAP/README.md

```diff
@@ -0,0 +1,30 @@
+# FreydCategoriesForCAP
+
+A package for various universal constructions mostly in an Ab-enriched context.
+
+This package includes implementations (amongst others) of:
+- [Freyd categories](#freyd-categories)
```

Comment:
I would also mention the additive closure here.

Reply:
Good idea, thanks.